app/views/questionnaire.py

```diff
@@ -222,21 +219,33 @@ def post_expire_session(eq_id, form_type, collection_id):  # pylint: disable=unu
 def get_session_expired(eq_id, form_type, collection_id):  # pylint: disable=unused-argument
     schema = load_schema_from_params(eq_id, form_type)
     return render_theme_template(schema['theme'], template_name='session-expired.html',
-                                 analytics_ua_id=settings.EQ_UA_ID,
+                                 analytics_ua_id=current_app.config['EQ_UA_ID'],
                                  survey_title=TemplateRenderer.safe_content(schema['title']))
 
 
-def submit_answers(eq_id, form_type, collection_id, metadata, answer_store):
+def submit_answers(eq_id, form_type, collection_id, metadata, answer_store, submitter):
     is_completed, invalid_location = is_survey_completed(answer_store, metadata)
 
     if is_completed:
         path_finder = PathFinder(g.schema_json, answer_store, metadata)
-        submitter = SubmitterFactory.get_submitter()
+
+        encrypter = Encrypter(
+            current_app.config['EQ_SUBMISSION_SR_PRIVATE_SIGNING_KEY'],
+            current_app.config['EQ_SUBMISSION_SR_PRIVATE_SIGNING_KEY_PASSWORD'],
+            current_app.config['EQ_SUBMISSION_SDX_PUBLIC_KEY'],
+        )
+
         message = convert_answers(metadata, g.schema_json, answer_store, path_finder.get_routing_path())
-        submitter.send_answers(message)
+        message = encrypter.encrypt(message)
```

Comment:
Yes, I've updated the tests to decrypt the message when they need to.

I take your point though, is this something we want to switch off in some circumstances?